packages/jest.config.js

```diff
@@ -0,0 +1,201 @@
+/*
```

Comment:
@takeutak Is this change intended to be in #444 or #442 ?

cc: @sfuji822 @hartm @jonathan-m-hamilton

Reply:
Clarified over the maintainer's call: #442

Reply:
Thank you for your comment.
As I said on the maintainer call, jest.config.js should be included by PR #444 (not #442).  So I deleted this file on PR #442.

Reply:
Moreover, I answered your question on PR #442.

> petermetz 3 days ago Member
> @takeutak Is this file scoped for the entire project (e.g. all packages) or just a sub-set of packages?

Just a sub-set of packages.
Please refer the line 123 on jest.config.js. The range is defined as the following.
roots: [
  "<rootDir>/ledger-plugin",
  "<rootDir>/routing-interface"
],

https://github.com/hyperledger/cactus/pull/442/files/0a2d04f0d30f25791c554a57412a188dfc7a670e#diff-bdf78ae09a2d9e94fa7baf7fa0103c048873338937c22c1d89f4786b214e01deR123

Reply:
> Moreover, I answered your question on PR #442.
> 
> > petermetz 3 days ago Member
> > @takeutak Is this file scoped for the entire project (e.g. all packages) or just a sub-set of packages?
> 
> Just a sub-set of packages.
> Please refer the line 123 on jest.config.js. The range is defined as the following.
> 
> ```
> roots: [
>  "<rootDir>/ledger-plugin",
>  "<rootDir>/routing-interface"
> ],
> ```
> 
> https://github.com/hyperledger/cactus/pull/442/files/0a2d04f0d30f25791c554a57412a188dfc7a670e#diff-bdf78ae09a2d9e94fa7baf7fa0103c048873338937c22c1d89f4786b214e01deR123

@takeutak Thank you for the info! Follow-up questions: 
Do you intend to have these tests executed as part of the CI in the future?
If yes to the above, how would the coverage statistics work? Is there a way to aggregate the coverage statistics between this newly introduced test runner and the pre-existing one?

Reply:
Discussed offline, we'll merge everything to support the TAP protocol.
https://testanything.org/

cc: @sfuji822 @hartm @jonathan-m-hamilton @takeutak